imgparse/_version.py

```diff
@@ -1,3 +1,3 @@
 """Defines package version.  Parsed by setup.py and imported by __init__.py."""
 
-__version__ = "1.12.7"
+__version__ = "1.12.8"
```

Comment:
I would like to avoid having Henry change his versioning on the parallel PR. Also, this is a tiny update, so I avoided going to 1.13.0. @elliott-imhoff you tell me if you are okay with that.

Reply:
Good with it. I think Henry's PR is stale anyway